arcgis-ios-sdk-samples/Utility Network/Trace utility network/TraceUtilityNetworkViewController.swift

```diff
@@ -188,8 +214,12 @@ class ConnectedTraceViewController: UIViewController, AGSGeoViewTouchDelegate {
     
     // MARK: Perform Trace
     @IBAction func traceNetwork(_ sender: Any) {
-        SVProgressHUD.show(withStatus: "Running connected trace…")
-        let parameters = traceParameters
+        SVProgressHUD.show(withStatus: "Running \(traceType.name) trace…")
```

Comment:
Should the name be lowercased to match the previous behavior?

```suggestion
        SVProgressHUD.show(withStatus: "Running \(traceType.name.lowercased()) trace…")
```